hudi-aws/src/main/java/org/apache/hudi/aws/sync/AWSGlueCatalogSyncClient.java

```diff
@@ -160,7 +159,7 @@ public void addPartitionsToTable(String tableName, List<String> partitionsToAdd)
       Table table = getTable(awsGlue, databaseName, tableName);
       StorageDescriptor sd = table.storageDescriptor();
       List<PartitionInput> partitionInputs = partitionsToAdd.stream().map(partition -> {
-        String fullPartitionPath = FSUtils.getPartitionPath(getBasePath(), partition).toString();
+        String fullPartitionPath = FSUtils.getPartitionPathInPath(getBasePath(), partition).toString();
```

Comment:
rename: `extractPartitionPath()`

Reply:
This method constructs the absolute path based on the base path and partition path, so better to rename it to `constructAbsolutePath`.  Created HUDI-7628 as a follow-up